cockatrice/src/player.cpp

```diff
@@ -292,21 +310,31 @@ Player::Player(const ServerInfo_User &info, int _id, bool _local, bool _judge, T
         libraryMenu->addAction(aAlwaysRevealTopCard);
         libraryMenu->addAction(aAlwaysLookAtTopCard);
         libraryMenu->addSeparator();
-        libraryMenu->addAction(aMoveTopToPlay);
-        libraryMenu->addAction(aMoveTopToPlayFaceDown);
-        libraryMenu->addAction(aMoveTopCardToBottom);
-        libraryMenu->addAction(aMoveBottomCardToGrave);
-        libraryMenu->addSeparator();
-        libraryMenu->addAction(aMoveTopCardToGraveyard);
-        libraryMenu->addAction(aMoveTopCardToExile);
-        libraryMenu->addAction(aMoveTopCardsToGraveyard);
-        libraryMenu->addAction(aMoveTopCardsToExile);
+        topLibraryMenu = libraryMenu->addTearOffMenu(QString());
+        bottomLibraryMenu = libraryMenu->addTearOffMenu(QString());
         libraryMenu->addSeparator();
         libraryMenu->addAction(aOpenDeckInDeckEditor);
         deck->setMenu(libraryMenu, aDrawCard);
-    } else {
-        handMenu = nullptr;
-        libraryMenu = nullptr;
+
+        topLibraryMenu->addAction(aMoveTopToPlay);
+        topLibraryMenu->addAction(aMoveTopToPlayFaceDown);
+        topLibraryMenu->addAction(aMoveTopCardToBottom);
+        topLibraryMenu->addSeparator();
+        topLibraryMenu->addAction(aMoveTopCardToGraveyard);
+        topLibraryMenu->addAction(aMoveTopCardToExile);
+        topLibraryMenu->addAction(aMoveTopCardsToGraveyard);
+        topLibraryMenu->addAction(aMoveTopCardsToExile);
```

Comment:
Following the logic we use elsewhere

```suggestion
        topLibraryMenu->addAction(aMoveTopCardToGraveyard);
        topLibraryMenu->addAction(aMoveTopCardsToGraveyard);
        topLibraryMenu->addAction(aMoveTopCardToExile);
        topLibraryMenu->addAction(aMoveTopCardsToExile);
```